src/compas_viewer/__main__.py

```diff
@@ -0,0 +1,57 @@
+# ==========================================================================
+# python -m compas_viewer -f "file path or directory path"
+# ==========================================================================
+
+import argparse
+from os import listdir
+from os import path
+
+from compas import json_load
+from compas import json_loadz
+
+from compas_viewer import Viewer
+
+ap = argparse.ArgumentParser()
+ap.add_argument(
+    "-f",
+    "--files",
```

Comment:
the default behavior should be just one `file`. and `files` as an possible option.
When user want to serialise several geometries, they are encouraged to do
```
json_dump([geo1, geo2, geo3],"path")
```
Instead of saving several json files.